css/style.css

```diff
@@ -6,58 +6,58 @@
 }
 
 .wrapper {
-  margin: 50px auto;
-  width: 280px;
+  -moz-box-shadow:    0px 0px 8px rgba(0,0,0,0.3);
+  -webkit-box-shadow: 0px 0px 8px rgba(0,0,0,0.3);
   background: white;
   border-radius: 10px;
-  -webkit-box-shadow: 0px 0px 8px rgba(0,0,0,0.3);
-  -moz-box-shadow:    0px 0px 8px rgba(0,0,0,0.3);
```

Comment:
My personal preference is to group both vendor-prefixed and non-prefixed properties together, so that you don't accidentally update one without updating the rest. I don't think there's any real order within that group, but the general consensus is that the non-prefixed property should go last.